lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -261,7 +261,7 @@ def call(params)
                           fail "The client \"#{client_id}\" does not exist in data product \"#{data_product.data_product_id}\""
                         end
                         fail "Client #{client_id} does not have project." unless project
-                        puts "Project #{project.pid} of client #{client_id} will receive #{users.count} users"
+                        GoodData.logger.info("Project #{project.pid} of client #{client_id} will receive #{users.count} users")
```

Comment:
info: Calls 'gooddata.logger.info("project #{project.pid} of client #{client_id} will receive #{users.count} users")' 2 times (DuplicateMethodCall) (reek)